module_utils/network_lsr/argument_validator.py

```diff
@@ -688,13 +693,20 @@ def _validate_post(self, value, name, result):
         result["family"] = family
 
         gateway = result["gateway"]
+        type = result["type"]
```

Comment:
`type` is a builtin function, so it should not be assigned: https://docs.python.org/3/library/functions.html#type

It is strange that this is not reported by a python linter or code checker, for example:
https://pylint.pycqa.org/en/latest/user_guide/messages/warning/redefined-builtin.html

Reply:
This is not resolved.